concordium-consensus/src/Concordium/External.hs

```diff
@@ -715,7 +730,11 @@ receiveFinalizationRecord bptr genIndex msg msgLen = do
 -- |Handle receipt of a transaction.
 -- The possible return codes are @ResultSuccess@, @ResultSerializationFail@, @ResultDuplicate@,
 -- @ResultStale@, @ResultInvalid@, @ResultConsensusShutDown@, @ResultExpiryTooLate@, @ResultVerificationFailed@,
--- @ResultNonexistingSenderAccount@, @ResultDuplicateNonce@, @ResultNonceTooLarge@, @ResultTooLowEnergy@.
+-- @ResultNonexistingSenderAccount@, @ResultDuplicateNonce@, @ResultNonceTooLarge@, @ResultTooLowEnergy@,
+-- @ResultCredentialDeploymentInvalidRegistrationID@,
+-- @ResultCredentialDeploymentInvalidSignatures@,
+-- @ResultCredentialDeploymentInvalidKeys@, @ResultCredentialDeploymentInvalidIP@,
+-- @ResultCredentialDeploymentInvalidAR@, @ResultCredentialDeploymentExpired@
```

Comment:
- `ResultCredentialDeploymentInvalidRegistrationID`: should this be `ResultDuplicateAccountRegistrationID`?
- `ResultCredentialDeploymentInvalidKeys` is not defined

Reply:
Yes, you're right.